conf/report/report-template.html

```diff
@@ -33,7 +35,50 @@
           {% for tool in report %}
           { "orderDataType": "test-status" },
           {% endfor %}
-        ]
+        ],
+          initComplete: function() {
+            this.api().columns([2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17]).every(function() {
+                var column = this;
+                var select = $('<select><option value="Select"></option></select>')
```

Comment:
General question to consider: instead of having in-line implementation of functions, putting them in a separate JS file might be easier to maintain in the long term.
It is easy to accumulate important code that is hard to understand if buried in a large HTML template (not sure what are good practices are to break this up, as it seems that the closure takes a few values from the context (such as `$(this).val()`) - so possibly the function to be implemented externally needs explicit parameters; but that is good for documentation).

So while this looks not a large problem here at this point (and maybe the conclusion might be that this particular use might be benign), it might become some problem in the future. So worthwhile considering a general plan for code structuring.